csb/src/main/scala/edu/msstate/dasi/csb/Neo4jWorkload.scala

```diff
@@ -257,12 +257,12 @@ object Neo4jWorkload extends Workload {
    * @return Graph containing the betweenness double values
    */
   def betweennessCentrality[VD: ClassTag, ED: ClassTag](graph: Graph[VD, ED], k: Int): Unit = {
-    val query = s"MATCH (n), pthroughn = shortestPath((a)-[*..$k]->(b)) " +
-      "WHERE n IN nodes(pthroughn) AND n <> a AND n <> b AND a <> b " +
-      "WITH n,a,b,count(pthroughn) AS sum " +
-      s"MATCH p = shortestPath((a)-[*..$k]->(b)) " +
-      "WITH n, a, b, tofloat(sumn)/ tofloat(count(p)) AS fraction " +
-      "RETURN n, sum(fraction);"
+    val query = s"MATCH (n), pthroughn = shortestPath((a)-[*]->(b))" +
```

Comment:
Why did you replace `[*..$k]` with `[*]`? This way the iterations are not bounded.